proxy/config/struct.go

```diff
@@ -63,7 +63,6 @@ type Plugin struct {
 
 //Admin has attributes for enabling, serverURI and metrics for admin data
 type Admin struct {
-	Enable           bool   `yaml:"enable"`
-	ServerURI        string `yaml:"serverUri"`
-	GoRuntimeMetrics bool   `yaml:"goRuntimeMetrics"`
```

Comment:
Can I know why do we remove this goRuntimeMetrics setting?

Reply:
it is conflict with go chassis config, I have to remove it and keep it enable by default, so that user will not be aware of this change, and I also remove this config in docs